tests/test_coded_location.py

```diff
@@ -6,7 +6,7 @@
 import random
 
 GRID_02 = load_grid('NZ_0_2_NB_1_1')
-LOCS = [CodedLocation(loc[0], loc[1], 0.001) for loc in GRID_02[20:50]]
+LOCS = [CodedLocation(loc[0], loc[1], 0.001) for loc in GRID_02[20:50]]  # type: ignore
```

Comment:
I think the problem here is that the `load_grid` function has a `Tuple[float, float]` return type signature where it needs to have a `list[Tuple[float, float]]`.

Reply:
To be fixed in type-fixing issue.